cfgov/jinja2/v1/_includes/organisms/email-signup-form.html

```diff
@@ -0,0 +1,44 @@
+{# ==========================================================================
+
+   email-subscribe-form.render()
+
+   ==========================================================================
+
+   Description:
+
+   Creates an email subscription form when given:
+
+   gd_code:       A GovDelivery code for a specified mailing list.
+
+   settings:      An object used to customize the markup.
+
+   settings.text: The text used used within the description markup.
+
+   ========================================================================== #}
+
+{% macro render(settings, gd_code='') %}
+{% import 'molecules/form-field-with-button.html' as form_field_with_button %}
+{% import 'molecules/_vars-content.html' as content %}
+<form class="o-email-subscribe-form"
```

Comment:
Mix of names of `email-subscribe-form` and `email-signup-form` between CSS naming and the file name.